pkg/controller/installation/core_controller.go

```diff
@@ -1182,6 +1188,16 @@ func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile
 		}
 	}
 
+	// Fetch any existing default BGPConfiguration object.
+	bgpConfiguration := &crdv1.BGPConfiguration{}
+	err = r.client.Get(ctx, types.NamespacedName{Name: "default"}, bgpConfiguration)
+	if err != nil && !apierrors.IsNotFound(err) {
+		r.SetDegraded("Unable to read BGPConfiguration", err, reqLogger)
+		return reconcile.Result{}, err
+	}
+
+	// TODO , any default settings need to be added on bgpconfig object and update back ?
```

Comment:
Nope, no need to write anything back for now. Can just remove this TODO.

Reply:
ok .